<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HSVDialog</class>
 <widget class="QDialog" name="HSVDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>253</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>HSV color setting</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QGroupBox" name="centerGroupBox">
       <property name="title">
        <string>Select the reference point </string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="3" column="3">
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="4">
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="5">
         <widget class="QDoubleSpinBox" name="sat_first">
          <property name="prefix">
           <string notr="true">SAT (0-100%) : </string>
          </property>
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>255.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QDoubleSpinBox" name="red">
          <property name="prefix">
           <string notr="true">RED     (0-255) : </string>
          </property>
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>255.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QDoubleSpinBox" name="blue">
          <property name="prefix">
           <string notr="true">BLUE    (0-255) : </string>
          </property>
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>255.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <spacer name="verticalSpacer_9">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="4">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="3">
         <widget class="QDoubleSpinBox" name="green">
          <property name="prefix">
           <string notr="true">GREEN (0-255) : </string>
          </property>
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>255.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="5">
         <widget class="QDoubleSpinBox" name="hue_first">
          <property name="prefix">
           <string notr="true">HUE  (0-360Â°) : </string>
          </property>
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>255.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="2">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <widget class="QToolButton" name="pointPickingButton_first">
          <property name="toolTip">
           <string>Pick the plane center (click again to cancel)</string>
          </property>
          <property name="icon">
           <iconset resource="../../../../qCC/icons.qrc">
            <normaloff>:/CC/images/ccPointPicking.png</normaloff>:/CC/images/ccPointPicking.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="5">
         <widget class="QDoubleSpinBox" name="val_first">
          <property name="prefix">
           <string notr="true">VAL (0-100%) : </string>
          </property>
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>255.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="horizontalGroupBox">
     <property name="title">
      <string>Choose which points to keep </string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QRadioButton" name="retain">
        <property name="text">
         <string>Retain</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QRadioButton" name="exclude">
        <property name="text">
         <string>Exclude</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QRadioButton" name="both">
        <property name="text">
         <string>Both</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../qCC/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HSVDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HSVDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
